src/axom/quest/IntersectionShaper.hpp

```diff
@@ -705,19 +693,20 @@ class IntersectionShaper : public Shaper
     this->getDC()->RegisterField(volFracName, volFrac);
 
     // Initialize hexahedral elements
-    m_hexes = axom::Array<HexahedronType>(NE, NE);
-    axom::ArrayView<HexahedronType> hexes_view = m_hexes.view();
+    m_hexes = axom::Array<HexahedronType>(NE, NE, kernel_allocator);
```

Comment:
Should we be toggling between `kernel` and `device` in our naming?

Reply:
Good catch! Changed `kernel_allocator` to `device_allocator` to keep naming standard.